src/sentry/utils/meta.py

```diff
@@ -1,4 +1,14 @@
 from collections.abc import Mapping
+from typing import List, Literal, Optional, TypedDict, Union
+
+RemarkType = Literal["a", "x", "s", "m", "p", "e"]
+
+
+class Remark(TypedDict):
+    rule_id: str
+    ty: RemarkType
+    range_start: Optional[int]
+    range_end: Optional[int]
```

Comment:
This range is in _bytes_, not UTF8 codepoints.

Reply:
Made these changes in a follow up PR - will point them out there.